docs/migration-to-NRI.md

```diff
@@ -0,0 +1,227 @@
+
+- For instance, an explicit type annotation for the balloons policy, which used to be:
+  ```yaml
+  ...
+  metadata:
+    annotations:
+      balloon.balloons.cri-resource-manager.intel.com/container.$CONTAINER_NAME: $BALLOON_TYPE`
+  ...
+  ```
+
+- Should now be:
+  ```yaml
+  ...
+  metadata:
+    annotations:
+      balloon.balloons.nri-resmgr.intel.com/container.$CONTAINER_NAME: $BALLOON_TYPE`
+  ...
+  ```
+
+- Similarly a workload opt-out annotation from exclusive CPU allocation for the topology-aware policy, which used to be:
+  ```yaml
+  ...
+  metadata:
+    annotations:
+      prefer-shared-cpus.cri-resource-manager.intel.com/container.$CONTAINER_NAME: "true"
+  ...
+  ```
+
+- Should now be:
+  ```yaml
+  ...
+  metadata:
+    annotations:
+      prefer-shared-cpus.nri-resmgr.intel.com/container.$CONTAINER_NAME: "true"
+  ...
+  ```
+
+- Similar changes are needed for any cri-resmgr-specific annotation that uses the same semantic scoping for key syntax.
+
+
+All of the annotations:
+| Was                                                 | Is now                                    |
+| --------------------------------------------------- | ----------------------------------------- |
+| cri-resource-manager.intel.com/afffinity            | nri-resmgr.intel.com/affinity             |
+| cri-resource-manager.intel.com/anti-afffinity       | nri-resmgr.intel.com/anti-affinity        |
+| cri-resource-manager.intel.com/prefer-isolated-cpus | nri-resmgr.intel.com/prefer-isolated-cpus |
+| cri-resource-manager.intel.com/prefer-shared-cpus   | nri-resmgr.intel.com/prefer-shared-cpus   |
+| cri-resource-manager.intel.com/cold-start           | nri-resmgr.intel.com/cold-start           |
+| cri-resource-manager.intel.com/memory-type          | nri-resmgr.intel.com/ memory-type         |
+| prefer-isolated-cpus.cri-resource-manager.intel.com | prefer-isolated-cpus.nri-resmgr.intel.com |
+| prefer-shared-cpus.cri-resource-manager.intel.com   | prefer-shared-cpus.nri-resmgr.intel.com   |
+| memory-type.cri-resource-manager.intel.com          | memory-type.nri-resmgr.intel.com          |
+| cold-start.cri-resource-manager.intel.com           | cold-start.nri-resmgr.intel.com           |
+| prefer-reserved-cpus.cri-resource-manager.intel.com | prefer-reserved-cpus.nri-resmgr.intel.com |
+| rdtclass.cri-resource-manager.intel.com             | rdtclass.nri-resmgr.intel.com             |
+| blockioclass.cri-resource-manager.intel.com         | blockioclass.nri-resmgr.intel.com         |
+| toptierlimit.cri-resource-manager.intel.com         | toptierlimit.nri-resmgr.intel.com         |
+| topologyhints.cri-resource-manager.intel.com        | topologyhints.nri-resmgr.intel.com        |
+| balloon.balloons.cri-resource-manager.intel.com     | balloon.balloons.nri-resmgr.intel.com     |
```

Comment:
In commit bdbb668186a1ab244aca166564bea1c5bfd7c147 @jukkar updated these from `nri-resource-policy.intel.com` to `resource-policy.nri.io`. We need to update this table accordingly.